### PR TITLE
[ADDED] Web Enhancing JavaScripts Added

### PR DESCRIPTION
**Pull request by Mahitej28**
# Description

- Added Javascript scripts for `Typewriter_Animation` and `Image_Slider` 
- Typewriter Animation script creates a typing effect where text is gradually displayed on the screen as if it's being typed. 
- Image Slider script creates an image slider that automatically cycles through a set of images.

## Fixes:  #1437 

This PR is made in reference to Issue No. #1437 

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation Update

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by Mahitej28** [comment]
@ClasherKasten kindly have a look at this PR, as it has been a long time since I created this PR

File: JavaScript/Typewriter_Animation/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Typing Animation using JavaScript
+
+This script creates a typing effect where text is gradually displayed on the screen, as if it's being typed. 
+It is used to make websites more engaging.
```
Comment (ClasherKasten) [inline JavaScript/Typewriter_Animation/README.md:4]: Looks more like a terminal thing -- at least I don't see anything there that could make **websites** more engaging.

**Review by ClasherKasten — CHANGES_REQUESTED** [review]
It would be a better experience if the buttons in the image slider don't shift around when the image height changes.

File: JavaScript/Typewriter_Animation/README.md (hunk 1 above)
Comment (Mahitej28) [reply JavaScript/Typewriter_Animation/README.md:4]: > Looks more like a terminal thing -- at least I don't see anything in the code that could make **websites** more engaging.

Dynamic Typing makes websites more attractive users can integrate it with their HTML CSS Code to see in websites

**Comment by Mahitej28** [comment]
> It would be a better experience if the buttons in the image slider don't shift around when the image height changes.

Sure I would work upon that

File: JavaScript/Typewriter_Animation/README.md (hunk 1 above)
Comment (Mahitej28) [reply JavaScript/Typewriter_Animation/README.md:4]: > > Looks more like a terminal thing -- at least I don't see anything in the code that could make **websites** more engaging.
> 
> Dynamic Typing makes websites more attractive users can integrate it with their HTML CSS Code to see in websites

If you want I will integrate it HTML and CSS

**Comment by Mahitej28** [comment]
@ClasherKasten all the required changes done you can go through it and let me know

**Comment by Mahitej28** [comment]
@ClasherKasten you can have a look at it and let me know if any changes to be done, also if its correct do assign a level of PR under GSSo'C 23

**Comment by Mahitej28** [comment]
Its been a long time since I posted this PR, it would be great if you go through it

**Comment by Mahitej28** [comment]
@ClasherKasten it's really been a long time since I created a pull request, according to me the project should be timely updated in github either u merge this pull request or ask for more changes just keeping it as it is doesn't make any sense

**Review by ClasherKasten — APPROVED** [review]
LGTM

**Comment by Mahitej28** [comment]
@ClasherKasten can u give it a GSSoC ' 23 label as I am currently a contributor there, it would be great

**Comment by Mahitej28** [comment]
@ClasherKasten please change the GSSOC ' 23 label to gssoc23 as my leaderboard at the portal is not getting updated with this pr

**Pull request merged**